src/Raven.Client/Documents/Commands/GetDocumentsCommand.cs

```diff
@@ -33,20 +35,22 @@ public GetDocumentsCommand(int start, int pageSize)
             _pageSize = pageSize;
         }
 
-        public GetDocumentsCommand(string id, string[] includes, bool metadataOnly)
+        public GetDocumentsCommand(string id, string[] includes, string[] counters, bool metadataOnly)
```

Comment:
This brakes the source-level compatibility. You need to add a separate constructor with that.

Reply:
Assert in this new ctor if this is not null.

Reply:
done